components/request-card/script.js

```
@@ -108,6 +108,270 @@
+  accordionContainer.addEventListener('click', function () {
+    isExtensionRequest ? renderLogs(data.id) : toggleAccordionPanel(panel);
+  });
+  const cardFooter = createElement({ type: 'div' });
+  cardFooter.appendChild(cardAssigneeButtonContainer);
+  cardFooter.appendChild(accordionContainer);
+  formContainer.appendChild(cardFooter);
+  rootElement.appendChild(formContainer);
+
+  formContainer.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    let formData = getFormEntries(new FormData(e.target));
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'formData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)